test/harvest/test_dimensions.py

```diff
@@ -13,9 +12,6 @@
 
 dotenv.load_dotenv()
 
-dimensions_user = os.environ.get("AIRFLOW_VAR_DIMENSIONS_API_USER")
-dimensions_password = os.environ.get("AIRFLOW_VAR_DIMENSIONS_API_PASS")
-
```

Comment:
not used in tests?

Reply:
We do use it in tests.

Reply:
i didn't see it referenced anywhere and tests pass without it -- i think the client is configured via env variables?

Reply:
yeah, looks like the login function itself pulls from the env vars directly in the `dimcli.login` call